pkg/minikube/bootstrapper/images/images.go

```diff
@@ -90,10 +144,12 @@ func etcd(v semver.Version, mirror string) string {
 	// Should match `DefaultEtcdVersion` in:
 	// https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/constants/constants.go
 	ev := "3.5.0-0"
-	majorMinorVersion := fmt.Sprintf("v%d.%d", v.Major, v.Minor)
 	imageName := "etcd"
+	majorMinorVersion := fmt.Sprintf("v%d.%d", v.Major, v.Minor)
 	if eVersion, ok := constants.KubeadmImages[majorMinorVersion][imageName]; ok {
 		ev = eVersion
+	} else {
+		ev = findLatestTagFromRepository(fmt.Sprintf(tagURLTemplate, kubernetesRepo(mirror), ev), ev)
```

Comment:
```suggestion
		ev = findLatestTagFromRepository(fmt.Sprintf(tagURLTemplate, kubernetesRepo(mirror), imageName), ev)
```
The version is getting passed in instead of the name

Reply:
Oh wow.. nice catch @spowelljr!